socorro/unittest/external/es/base.py

```diff
@@ -774,6 +794,15 @@ def test_with_version(self):
     return decorated
 
 
+class SuperSearchWithFields(SuperSearch):
```

Comment:
actually, you might not need to do `self.supersearch =` here. Just `supersearch = `